src/Core/src/Platform/Windows/KeyboardAcceleratorExtensions.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.System;
+
+namespace Microsoft.Maui.Platform
+{
+	public static class KeyboardAcceleratorExtensions
+	{
+		public static void UpdateAccelerator(this MenuFlyoutItemBase platformView, IMenuFlyoutItem menuFlyoutItem)
+		{
+			var keyboardAccelerators = menuFlyoutItem.Accelerators?[0].ToPlatform();
+
+			if (keyboardAccelerators is null)
+				return;
+
+			foreach (var keyboardAccelerator in keyboardAccelerators)
+				platformView.KeyboardAccelerators.Add(keyboardAccelerator);
+		}
+
+		// Single key (A, Delete, F2, Spacebar, Esc, Multimedia Key) accelerators and multi-key
+		// accelerators (Ctrl+Shift+M) are supported.
+		// Gamepad virtual keys are not supported.
+		public static IList<KeyboardAccelerator>? ToPlatform(this IAccelerator accelerator)
+		{
+			if (accelerator is null)
+				return null;
+
+			List<KeyboardAccelerator> result = new List<KeyboardAccelerator>();
+			
+			var key = accelerator.Key;
+			var modifiers = accelerator.Modifiers;
+
+			if (modifiers is not null)
+			{
+				var modifiersCount = modifiers.Count();
+				bool hasModifierMask = modifiersCount > 0;
+
+				if (hasModifierMask)
+				{
+					if (modifiersCount == 1)
+						result.Add(CreateSingleKeyAccelerator(modifiers.ElementAt(0), key));
+					else
+						result.Add(CreateMultiKeyAccelerator(modifiers, key));
+				}
+			}
+			else
+				result.Add(CreateSingleKeyAccelerator(string.Empty, key));
```

Comment:
```suggestion
			var keyboardAccelerator = new KeyboardAccelerator();
			keyboardAccelerator.Key = key.ToVirtualKey();

			if (modifiers is not null)
			{
				foreach (var mod in modifiers)
				{
					keyboardAccelerator.Modifiers |= mod .ToVirtualKeyModifiers();
				}
			}

			result.Add(keyboardAccelerator);
```

Reply:
added for now, and it actually fixed a new issue I was seeing so yay!